app/src/main/java/com/github/se/cyrcle/ui/map/overlay/ZoomControls.kt

```diff
@@ -18,10 +18,11 @@ import androidx.compose.ui.platform.testTag
 import androidx.compose.ui.unit.dp
 
 @Composable
-fun ZoomControls(onZoomIn: () -> Unit, onZoomOut: () -> Unit) {
+fun ZoomControls(modifier: Modifier = Modifier, onZoomIn: () -> Unit, onZoomOut: () -> Unit) {
```

Comment:
I think you can remove the type `Modifier`  here since you are already declaring the value